source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -252,6 +252,19 @@ def enumerate(podList = nil)
               telemetryProperties["DATA_COLLECTION_INTERVAL_MINUTES"] = @run_interval / 60
             end
           end
+          begin
+            if Dir.exist?("/etc/mdsd.d/config-cache/configchunks")
+              Dir.glob("/etc/mdsd.d/config-cache/configchunks/*.json") { |file|
+                if File.file?(file) && File.exist?(file)
+                  if File.foreach(file).grep(/LINUX_SYSLOGS_BLOB/).any?
+                    telemetryProperties["syslogEnabled"] = "true"
+                  end
+                end
+              }
+            end
+          rescue => errorStr
+            $log.warn("ApplicationInsights:: Exception in getting syslog status: #{errorStr}")
```

Comment:
Should the message be ApplicationInsights if we have moved the telemetry to kubepodinventory?